.env

```diff
@@ -0,0 +1 @@
+ATLAS_URI=mongodb+srv://UserCapstone:user123@cluster0.bhnwc.mongodb.net/myFirstDatabase?retryWrites=true&w=majority
```

Comment:
🔴 Please add your `.env`to your `.gitignore` 🔴 

If the `.env` is not being ignored you maybe have to delete the cache first.
See: https://stackoverflow.com/questions/25436312/gitignore-not-working

Reply:
Thank you! I've just choosen the wrong .gitignore...